docs/reference/components/sources/kubernetes_logs.cue

```diff
@@ -170,8 +170,8 @@ components: sources: kubernetes_logs: {
 				"""
 			required: false
 			type: array: {
-				default: []
-				items: type: string: examples: ["metadata.name!=pod-name-to-exclude"]
+				default: ""
+				items: type: string: examples: "metadata.name!=pod-name-to-exclude,metadata.name=mypod"
```

Comment:
```suggestion
				default: ""
				examples: ["metadata.name!=pod-name-to-exclude,metadata.name=mypod"]
```

Reply:
So in this case, this entry is specifically a string and not an array - I'm assuming these square brackets are a cue notation thing?

Reply:
Yes, and we can pass more than one example - which I suggest we do